pkg/asset/ignition/bootstrap/content/report.go

```diff
@@ -0,0 +1,56 @@
+package content
+
+const (
+	// ReportSystemdContents is a service that reports the bootstrap progress
+	// via a ConfigMap.
+	ReportSystemdContents = `
+[Unit]
+Description=Report the completion of the cluster bootstrap process
+# Workaround for https://github.com/systemd/systemd/issues/1312
+Wants=bootkube.service tectonic.service
+After=bootkube.service tectonic.service
+
+[Service]
+# Workaround for https://github.com/systemd/systemd/issues/1312 and https://github.com/opencontainers/runc/pull/1807
+ExecStartPre=/usr/bin/test -f /opt/tectonic/.bootkube.done
+ExecStartPre=/usr/bin/test -f /opt/tectonic/.tectonic.done
+ExecStart=/opt/tectonic/report-progress.sh /opt/tectonic/auth/kubeconfig
+
+Restart=on-failure
+RestartSec=5s
+
+[Install]
+WantedBy=multi-user.target
+`
+
+	// ReportShFileContents is a script for reporting the bootstrap progress.
+	ReportShFileContents = `#!/usr/bin/env bash
+set -e
+
+KUBECONFIG="$1"
+
+echo "Reporting install progress..."
+
+manifest=$(mktemp)
+trap "rm ${manifest}" EXIT
+
+cat <<EOF > "${manifest}"
+apiVersion: v1
+kind: Event
+metadata:
+  name: bootstrap-complete
+  namespace: kube-system
+involvedObject:
+  namespace: kube-system
+message: cluster bootstrapping has completed
+firstTimestamp: $(date --rfc-3339=seconds | tr ' ' 'T')
```

Comment:
I'd rather stick to [POSIX][1] and use:

```sh
date -u +'%Y-%m-%dT%H:%M:%SZ'
```

which will also match [Go's `RFC3339` format][2] ([play][3]).

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/date.html
[2]: https://golang.org/pkg/time/#pkg-constants
[3]: https://play.golang.org/p/8XRSSO_-WQD